httpbin/core.py

```diff
@@ -375,6 +375,29 @@ def set_cookies():
     return r
 
 
+@app.route('/cookies/set', methods=['PUT'])
```

Comment:
Should allow overwriting of cookies on the response (for cookies with the same "key")